### PR TITLE
Now using getManyExpressions instead of one expression at a time for loading chat messages

### PR DESCRIPTION
**Pull request by fayeed**
The PR uses ad4m version 0.1.16 and some things have not migrated yet to use the new API. Also, Graphql tests are broken because the new version doesn't generate the correct type schema in the correct file, I have raised a PR here: https://github.com/perspect3vism/ad4m/pull/22

### CODE REVIEW

File: src/store/data/actions/loadExpressions.ts
Comment (jdeepee) [inline src/store/data/actions/loadExpressions.ts:94]: Will this map fail if the above `linkQuery.filter()` returns an `ExpressionRef` and not a `LinkExpression`?

**Review by jdeepee — CHANGES_REQUESTED** [review]

File: src/store/data/actions/loadExpressions.ts
Comment (fayeed) [reply src/store/data/actions/loadExpressions.ts:94]: Can the linkworker return an ExpressionRef instead of LinkExpression?

Comment (jdeepee) [reply src/store/data/actions/loadExpressions.ts:94]: lol code reading error on my side, looks good 👍

**Review by jdeepee — APPROVED** [review]

**Comment by jdeepee** [comment]
Lets wait for @leifriksheim review before merging

File: src/store/data/actions/loadExpressions.ts
Comment (fayeed) [reply src/store/data/actions/loadExpressions.ts:94]: Ok cool 😅

**Review by leifriksheim — APPROVED** [review]
@fayeed I will implement this into the chat-view web component as we will start using this component soon. But let's merge it and have it for now until we are using the chat-view.

**Pull request merged**
